include/netlink/route/link/bonding.h

```diff
@@ -12,6 +12,9 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+extern void rtnl_link_set_bondmode(struct rtnl_link *, uint8_t);
```

Comment:
API needs to be added to `libnl-route-3.sym` (at the `libnl_3_8` section at bottom of the file)